react_native/src/screens/root_screens/SplashScreenComponent.js

```diff
@@ -1,119 +1,136 @@
-                        buttonColor={"#000000"}
-                        width={261}
-                        height={54}
-                        bold={true}
-                        onPress={() => {}}
-                    /> : null}
-                </View>
-            </SafeAreaView>
-        </Animatable.View>
-    );
+        <View style={styles.registrationOA2}>
+          <ActionButtonComponent
+            logo={googleLogo}
+            logoSize={{ width: 24, height: 24 }}
+            label={"Continue with Google"}
+            labelColor={"rgba(0,0,0,0.55)"}
+            buttonColor={"#ffffff"}
+            width={responsiveScreenWidth(80)}
+            height={54}
+            bold={true}
+            onPress={() => {}}
+          />
+          <View style={{ padding: 8 }} />
+          {Platform.OS === "ios" ? (
+            <ActionButtonComponent
+              logo={appleLogo}
+              logoSize={{ width: 24, height: 24 }}
+              label={"Continue with Apple"}
+              labelColor={"#ffffff"}
+              buttonColor={"#000000"}
+              width={responsiveScreenWidth(80)}
+              height={54}
+              bold={true}
+              onPress={() => {}}
+            />
+          ) : null}
+        </View>
+      </SafeAreaView>
+    </Animatable.View>
+  );
 }
 
 const styles = StyleSheet.create({
-    container: {
-        paddingTop: "71%",
-        justifyContent: "center",
-    },
-    title: {
-        position: 'relative',
-        top: '36%',
-        left: '30%',
-        right: '50%',
-        width: 234,
-        height: 74
-    },
-    registration: {
-        flex: 1,
-        flexDirection: "row",
-        paddingTop: "50%",
-        paddingRight: "8%",
-        alignItems: "flex-end",
-        justifyContent: "flex-end"
-    },
-    registrationOA2: {
-        flex: 1,
-        flexDirection: "column",
-        paddingTop: Platform.OS === 'ios' ? "60%" : "30%",
-        paddingRight: "8%",
-        alignItems: "flex-end",
-        justifyContent: "flex-end"
-    }
-})
+  container: {
+    paddingTop: "71%",
+    justifyContent: "center",
+    // backgroundColor: "#ffffff",
+  },
+  title: {
+    top: responsiveScreenHeight(5),
+    left: responsiveScreenWidth(20),
+    width: 234,
+    height: 74,
+    // backgroundColor: "blue",
+  },
+  tuter: {
+    color: "white",
+    fontSize: responsiveScreenFontSize(8),
+    textShadowColor: "rgba(0, 0, 0, 0.75)",
+    textShadowOffset: { width: 0, height: 3 },
+    textShadowRadius: 10,
+    position: "absolute",
+  },
+  logo: {
+    marginTop: responsiveScreenHeight(8),
+    marginLeft: responsiveScreenWidth(52),
+    height: 24,
+    width: 24,
+    // backgroundColor: "green",
+  },
+  registration: {
+    flex: 1,
+    flexDirection: "row",
+    paddingTop: responsiveScreenHeight(10),
+    marginLeft: responsiveScreenWidth(-5),
+    // paddingRight: "8%",
+    alignItems: "flex-end",
+    // justifyContent: "flex-end",
```

Comment:
nit: remove these if not being used, same applies to other places that they are being used

Reply:
@NataliaLopez17